### DISCUSSION TITLE
Can I use this component library to develop browser plug-ins? Does it support Chrome, Edge, and other PC browser pages?

### DISCUSSION THREAD

**Discussion by solitudezhous** [hellof2e/quark-design#308] [Q&A]
My need is to develop a browser plugin that embeds the plugin's page in the form of shadow-dom on the side of PC browser pages. I haven't found a suitable UI component library. I recently learned about this component library based on WCs, and I want to know if this component library is available for both PC and mobile devices, and whether it supports PC browsers such as Chrome and Edge, and whether it can meet my needs？I'm looking forward to your response.

**Comment by xxxsf** [comment]
Yes, quarkd can run anywhere there is a browser.

**Reply by xxxsf** [reply]
Except IE browser
